apps/docs-next/src/content/learn/examples/animation/transitions.mdx

```diff
@@ -6,8 +6,29 @@ showInSidebar: false
 
 Transition seamlessly between GLTF animations.
 
-## How it works
+<Example path="animation/transitions" />
 
-The important stuff happens in the `transitionTo` function in `Character.svelte`
+## Explanation
 
-<Example path="animation/transitions" />
+GLTF is a comprehensive file format for 3D models, and it supports animations.
```

Comment:
I think the correct capitalization is "glTF" although I didn't follow through with this in the rest of the codebase. Still, it's never too late to start!